desktop/release-notes.md

```diff
@@ -71,12 +115,16 @@ For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/ge
 - You can now reclaim disk space more quickly when files are deleted in containers. Related to [docker/for-mac#371](https://github.com/docker/for-mac/issues/371).
 - Fixed a bug that prevented containers accessing 169.254.0.0/16 IPs. Fixes [docker/for-mac#6825](https://github.com/docker/for-mac/issues/6825).
 - Fixed a bug in `com.docker.diagnose check` where it would complain about a missing vpnkit even when vpnkit is not expected to be running. Related to [docker/for-mac#6825](https://github.com/docker/for-mac/issues/6825).
-
+- Fixed a bug where extension menu wasn't clickable. Fixes [docker/for-mac#6840](https://github.com/docker/for-mac/issues/6840) and [docker/for-mac#6855](https://github.com/docker/for-mac/issues/6855)
```

Comment:
Are these really forgotten fixes and features we missed in `4.20.1` release notes? 😬

Reply:
No, I misplaced the release note!